installer_scripts/install-yb-voyager

```diff
@@ -832,6 +835,25 @@ create_gather_assessment_metadata_dir() {
 	fi
 }
 
+create_guardrail_scripts_dir() {
+	scripts_parent_dir="/opt/yb-voyager"
+	script_dir_path="guardrail_scripts"
+	scripts_dir_name="guardrail-scripts"
+	sudo mkdir -p $scripts_parent_dir/$scripts_dir_name
+
+	output "Installing the latest guardrail scripts"
```

Comment:
this output should be inside the if/else both

Reply:
Sorry not clear. It will be printing before the if/else statement anyways?

Reply:
Yes, but it says "latest", which isn't always the case.

Reply:
just remove the word "latest" otherwise.

Reply:
Ah okay